### PR TITLE
setup + link in readme with docs

### PR DESCRIPTION
**Pull request by Guseyn**
Initial setup: #2

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>com.artipie</groupId>
+    <artifactId>ppom</artifactId>
+    <version>0.3</version>
```
Comment (g4s8) [inline pom.xml:29]: @Guseyn `0.3.1` was released this week

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    </developer>
+  </developers>
+  <scm>
+    <connection>scm:git:git://github.com/artipie/helm-adapter.git</connection>
+    <developerConnection>scm:git:ssh://github.com:artipie/helm-adapter.git</developerConnection>
+    <url>https://github.com/artipie/helm-adapter/tree/master</url>
+  </scm>
+  <dependencies>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
```
Comment (g4s8) [inline pom.xml:55]: @Guseyn it's not needed to include hamcrest and junit dependencies, it should be included automatically from parent pom

File: README.md [hunk 3]
```diff
@@ -8,4 +8,14 @@
 [![Hits-of-Code](https://hitsofcode.com/github/artipie/helm-adapter)](https://hitsofcode.com/view/github/artipie/helm-adapter)
 [![PDD status](http://www.0pdd.com/svg?name=artipie/helm-adapter)](http://www.0pdd.com/p?name=artipie/helm-adapter)
 
-Helm repository adapter.
+## Helm repository adapter.
+
+[Usage](https://helm.sh/docs/intro/using_helm/).
```
Comment (g4s8) [inline README.md:13]: @Guseyn please add more detauls about how this particular implementation of helm repository works. This link contains too many details which are not related to registry/repository. We need some essence from helm docs here.

File: src/main/java/com/artipie/helm/package-info.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/**
+ * Pypi files.
```
Comment (g4s8) [inline src/main/java/com/artipie/helm/package-info.java:26]: @Guseyn pypi?

File: src/test/java/com/artipie/helm/package-info.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+/**
+ * Pypi files.
```
Comment (g4s8) [inline src/test/java/com/artipie/helm/package-info.java:26]: @Guseyn pypi?

File: src/main/java/com/artipie/helm/package-info.java (hunk 4 above)
Comment (Guseyn) [reply src/main/java/com/artipie/helm/package-info.java]: @g4s8 fixed

File: src/test/java/com/artipie/helm/package-info.java (hunk 5 above)
Comment (Guseyn) [reply src/test/java/com/artipie/helm/package-info.java]: @g4s8 fixed

File: README.md (hunk 3 above)
Comment (Guseyn) [reply README.md]: @g4s8 improved

File: pom.xml (hunk 2 above)
Comment (Guseyn) [reply pom.xml]: @g4s8 fixed

File: pom.xml (hunk 1 above)
Comment (Guseyn) [reply pom.xml]: @g4s8 fixed

**Comment by Guseyn** [comment]
@g4s8 check the updates above

**Pull request merged**